apps/lead/tests/test_schemas.py

```diff
@@ -271,8 +271,6 @@ def test_lead_query_filter(self):
                 },
                 [lead5]
             ),
-            ({'hasAssessment': True}, [lead1, lead2]),
-            ({'hasAssessment': False}, [lead3, lead4, lead5]),
```

Comment:
Can't we fix this instead?

Reply:
@thenav56 , I have created Test Factories required for assessment registry in next PR https://github.com/the-deep/server/pull/1324. So this case will be handled in that PR.